poms/pom.xml

```diff
@@ -48,6 +48,7 @@
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+        <scm.gitBaseUrl>https://github.com/openhab</scm.gitBaseUrl>
```

Comment:
Does this have to be redefined or can it be re-used from the parent pom? Should there be a bidirectional relation between the poms...?

Reply:
Unfortunately, it has to be redefined as this pom has no parent. It's just built with openHAB-Core, but does not inherit from it. There ist also the idea to carve out this pom and make everything inherit from it, but one step after another. I'd like to start with the least invasive step.